wordsearch.js

```diff
@@ -0,0 +1,501 @@
+(function(){
```

Comment:
Please update this file to 2 space indentation + remove unnecessary blank lines.
Also, naming conventions can be updated to make it readable.